lnd.go

```diff
@@ -92,7 +91,7 @@ var (
 // lndMain is the true entry point for lnd. This function is required since
 // defers created in the top-level scope of a main method aren't executed if
 // os.Exit() is called.
-func lndMain() error {
+func LndMain() error {
```

Comment:
The comment should be upper case too. And the linter suggests to name this func `Main` because it is now in the package `lnd`.